client/src/Clusters/TopCharts/TopCharts.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from "react";
+import "./SingleAppContainer.js";
+import "./TopChartsStyle.css";
+import "../styles.css";
+import SingleAppContainer from "./SingleAppContainer.js";
+ 
+class TopCharts extends Component {
+  render() {
+    const makeColumn = item => (
+      <div>
```

Comment:
You can remove the outer div

Reply:
for some reason this div keeps it all together. when i remove it, the icons and words spread out all over